apps/web-portal/lib/use-cases/preview/PreviewList.mjs

```diff
@@ -0,0 +1,10 @@
+import {UseCase} from '@zorko-io/util-use-case'
+
+export class PreviewList extends UseCase {
+
```

Comment:
validation rules will be placed in usecase?

Reply:
sure, it's just an example, however, I'll drop todo here